test/core/util/grpc_fuzzer.bzl

```diff
@@ -33,3 +35,36 @@ def grpc_fuzzer(name, corpus, srcs = [], deps = [], data = [], size = "large", *
         }),
         **kwargs
     )
+
+def grpc_proto_fuzzer(name, corpus, proto, srcs = [], deps = [], data = [], size = "large", **kwargs):
+    proto_library(
+        name = name + "-proto",
```

Comment:
Nit: Conventionally, hidden targets like this have names prefixed with underscores. Also, what's the reasoning behind using a dash rather than an underscore before `proto`?

Reply:
The shift key on my laptop is a bit sticky...

Reply:
(done)